shells/tests/wdio.conf.js

```diff
@@ -144,6 +144,9 @@ exports.config = {
   // your test setup with almost no effort. Unlike plugins, they don't add new
   // commands. Instead, they hook themselves up into the test process.
   services: ['chromedriver'],
+  // Directory for logs: See
+  // https://webdriver.io/docs/wdio-chromedriver-service.html
+  outputDir: '.logs',
```

Comment:
Super minor nit: could we call this `.chromedriver-logs` to make it really obvious what it's for?

Reply:
Renaming to .wdio-logs as per our chat